### PR TITLE
News 5 articles

### PR DESCRIPTION
**Pull request by berri-puff**
error handling done previously with an invalid path, unsure what other error testings is available for this endpoint

### CODE REVIEW

File: controllers/article-controller.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+const { getArticleByID, selectAllArticles } = require("../models/article-model")
 
 exports.getsArticle = (req,res, next)=>{
     const {article_id} = req.params
     getArticleByID(article_id).then((foundArticle)=>{
         res.status(200).send({article: foundArticle})
     }).catch(next)
+    }
+
+    exports.getsAllArticles = (req,res,next) => {
```
Comment (itskatherine) [inline controllers/article-controller.js:10]: Clear function names here 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -38,61 +36,86 @@ describe("/api/topics", () => {
...
-    .then(({body})=>{
-      expect(body.msg).toBe('Bad Request')
-    })
-  })
-})
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+       const {articles} = response.body
+        expect(articles).toHaveLength(13);
```
Comment (itskatherine) [inline __tests__/app.test.js:105]: Great addition of a length check so you don't get a false positive with an empty array

File: __tests__/app.test.js [hunk 3]
```diff
@@ -38,61 +36,86 @@ describe("/api/topics", () => {
...
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            title: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+          });
+          expect(piece).hasOwnProperty('comment_count')
+        });
```
Comment (itskatherine) [inline __tests__/app.test.js:118]: You should also check the object don't have the `body` property, as these articles could have a key of body and still pass this test. 
Should also have a test that checks the comments are in the correct order (i.e. descending by date/created_at), I recommend `jest-sorted` for this

**Review by itskatherine — COMMENTED** [review]
Hi Jenny, once you've added the additional checks to your tests and got them passing, you're good to merge 👍

File: __tests__/app.test.js (hunk 3 above)
Comment (berri-puff) [reply __tests__/app.test.js:118]: Hi Kathrine, the sorted check is on line106

**Pull request merged**